src/linear_algebra/clifford_algebra/equivs.lean

```diff
@@ -89,3 +89,104 @@ alg_equiv.of_alg_hom to_complex
 attribute [protected] Q
 
 end clifford_algebra_complex
+
+/-! ### The clifford algebra isomorphic to the quaternions -/
+namespace clifford_algebra_quaternion
+
+open_locale quaternion
+open quaternion_algebra
+
+variables {R : Type*} [comm_ring R] (c₁ c₂ : R)
+
+/-- The quadratic form. -/
+def Q : quadratic_form R (R × R) :=
+c₁ • quadratic_form.lin_mul_lin (linear_map.fst _ _ _) (linear_map.fst _ _ _) +
+c₂ • quadratic_form.lin_mul_lin (linear_map.snd _ _ _) (linear_map.snd _ _ _)
```

Comment:
I don't see a better way, but this is quite a mouthful to talk about `c_1 x^2 + c_2 y^2`.

Also, should the name be a bit longer and descriptive? We can then have `Q` be a local notation in this file.

Reply:
This is consistent with how I handled the complex case above. My thinking behind the short name is that it's convenient, and the quadratic form isn't special beyond the fact that "it's the one we use in the equiv below".
I'm happy to rename this one and the complex one at the same time.


Note that while an `local notation` will make a long name more appealing when reading the file, the result in doc-gen will be ugly.

Regarding the value, it is indeed unfortunate how verbose it ends up, but doing it this way at least avoids proof obligations.

Reply:
Agreed on all points.

Reply:
Is the action then simply to improve the docstring of `Q` without renaming it?